drivers/driver.go

```diff
@@ -8,25 +8,28 @@ import (
 
 // RDMAVolumeDriver holds all the information pertaining to a RDMA Volume Driver.
 type RDMAVolumeDriver struct {
-	StorageController RDMAStorageController
+	StorageController StorageController
```

Comment:
This syntax looks strange to me, how do Go structs work? My main confusion stems from seeing `StorageController` twice in a row.